recipes/dj-static/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set package = "dj-static" %}
```

Comment:
New releases pulled from PyPI need a new `version and `md5`. This makes bumping versions easier.

I added the `package` so I can copy and paste the PyPI block between projects (I've had many errors here in the past).